test/extended/builds/pipeline_origin_bld.go

```diff
@@ -1,161 +1,42 @@
 package builds
 
 import (
-	"context"
 	"fmt"
-	"os"
-	"strings"
 	"time"
 
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
 
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/util/validation"
-	"k8s.io/apimachinery/pkg/util/wait"
-	e2e "k8s.io/kubernetes/test/e2e/framework"
-	"k8s.io/kubernetes/test/e2e/framework/pod"
-	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
-
-	buildv1 "github.com/openshift/api/build/v1"
 	exutil "github.com/openshift/origin/test/extended/util"
 	"github.com/openshift/origin/test/extended/util/jenkins"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
 )
 
-const (
-	localClientPluginSnapshotImageStream = "jenkins-client-plugin-snapshot-test"
-	localClientPluginSnapshotImage       = "openshift/" + localClientPluginSnapshotImageStream + ":latest"
-	localSyncPluginSnapshotImageStream   = "jenkins-sync-plugin-snapshot-test"
-	localSyncPluginSnapshotImage         = "openshift/" + localSyncPluginSnapshotImageStream + ":latest"
-	clientLicenseText                    = "About OpenShift Client Jenkins Plugin"
-	syncLicenseText                      = "About OpenShift Sync"
-	clientPluginName                     = "openshift-client"
-	syncPluginName                       = "openshift-sync"
-	secretName                           = "secret-to-credential"
-	secretCredentialSyncLabel            = "credential.sync.jenkins.openshift.io"
-	envVarsPipelineGitRepoBuildConfig    = "test-build-app-pipeline"
-)
-
-var _ = g.Describe("[sig-builds][Feature:Builds][sig-devex][Feature:Jenkins][Slow] openshift pipeline build", func() {
+var _ = g.Describe("[sig-builds][Feature:JenkinsRHELImagesOnly][Feature:Jenkins][Feature:Builds][sig-devex][Feature:Jenkins][Slow] openshift pipeline build", func() {
```

Comment:
combining all the existing jenkins related  `[Feature:...]` from both this file and the 2 pipeline_*.go files that are deleted here means that all the existing `openshift-tests` suites use this same test via the ginkgo searching mechanism